games/Hangman.py

```diff
@@ -0,0 +1,128 @@
+import random
+
+# Simple terminal hangman game with ASCII art
+def choose_random_word():
+    word_list = ["hangman", "python", "programming", "developer", "challenge", "knowledge"]
+    return random.choice(word_list)
+
+def display_word(word, guessed_letters):
+    """Chooses a random word between the ones listed in word_List
+        Returns: (str) the chosen word.
+    """
+    display = ''
+    for letter in word:
+        if letter in guessed_letters:
+            display += letter
+        else:
+            display += '_'
+    return display
+
+def draw_hangman(attempts):
+    """ Draws different stages of the Hangman game in ASCII art style,
+        based on the current number of attemps.
+    """
+    hangman_stages = [
+        """
+           ------
+           |    |
+           |
+           |
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |    |
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|\\
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|\\
+           |   /
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|\\
+           |   / \\
+           |
+        """
+    ]
+    
+    print(hangman_stages[attempts])
+
+def hangman():
+    word_to_guess = choose_random_word()
+    guessed_letters = []
+    attempts = 0
+    max_attempts = 6
+    
+    """Determine the current number of attempts and draw the hangman art.
```

Comment:
Trivial: can we move this to line 86 so it is directly under the function's definition?

Reply:
Everything else looks good to me! Will merge this PR in as soon as this is addressed! 🙂

Reply:
> 

Sure thing! It's moved now.